deploy/complete/docker-compose/docker-compose.yml

```diff
@@ -125,7 +129,7 @@ services:
     environment:
       - OADB_USER=orders_user
       - OADB_PW=Default_Password123#
-      - OADB_SERVICE=morders_tp
+      - OADB_SERVICE=mock
```

Comment:
This makes Orders run in mock mode, but the API server is not going to send any requests here since Users are mocked as well. When Users can run with a sqlite db, this will become relevant.

Reply:
Created CNS-166 to support users in mock mode too. Can you set the same var in `user` service for future compatibility?